mod/discussions/views/default/resources/discussion/owner.php

```diff
@@ -3,16 +3,22 @@
 $guid = elgg_extract('owner_guid', $vars);
 elgg_set_page_owner_guid($guid);
 
+elgg_entity_gatekeeper($guid);
 elgg_group_gatekeeper();
 
-$group = get_entity($guid);
-if (!elgg_instanceof($group, 'group')) {
+$container = get_entity($guid);
+$type = $container->type;
+$subtype = $container->getSubtype();
+if (!discussion_is_context_allowed($container->type, $container->getSubtype())) {
```

Comment:
Though we could leave this view limited to groups and a dev would have to override this resource view to allow user discussions.